src/utils/keyvalue.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2024 Renesas Electronics Corporation.
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <Poco/StringTokenizer.h>
```

Comment:
all such stuff should be done in `aos_core_common_cpp`. But we have to merge mending PR there first. And we may have other parsing helpers in future. So, the file should be named: `perser.cpp` or similar. And name space: `aos::utils::parser`.

Reply:
While aos_core_common_cpp is not merged, I renamed this file, put it's interfaces under aos::utils::parser.